vars/piperExecuteBin.groovy

```diff
@@ -48,6 +48,9 @@ void call(Map parameters = [:], String stepName, String metadataFile, List crede
                 echo "Context Config: ${config}"
             }
 
+            //Add ANS credential information to the config
+            config += ["ansServiceKeyCredentialsId": script.commonPipelineEnvironment.configuration.hooks.ansServiceKeyCredentialsId]
```

Comment:
If there is no hooks section in the config this will result in an NPE, so:
```suggestion
            config += ["ansServiceKeyCredentialsId": script.commonPipelineEnvironment.configuration.hooks?.ansServiceKeyCredentialsId]
```

Reply:
done with the other change